### DISCUSSION TITLE
📣 VSCode users please upgrade to version 1.55

### DISCUSSION THREAD

**Discussion by davecheney** [community/community#37300] [Copilot Conversations]
VSCode pilots, please upgrade to version 1.55 of the Copilot extension (or later)

_Note_: If you're running VSCode 1.67 or earlier, you must first upgrade VSCode to unlock newer versions of the Copilot extension.

**Comment by 3kh0** [comment]
will do!

**Reply by Qusai1995albreazet** [reply]
https://github.com/orgs/community/discussions/37300#discussion-4505180

**Comment by thiagorussi** [comment]
ok
